### PR TITLE
Fix permadiffs in regional `health_checks` and `backend_service`

### PR DESCRIPTION
**Pull request by upodroid**
Applying #4396 to regional equivalent

`log_config` is computed on `google_compute_backend_service` so doing the same on the regional one.

https://github.com/hashicorp/terraform-provider-google/blob/master/google/resource_compute_backend_service.go#L639

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: fixed a perma-diff on `google_compute_region_health_check` when `log_config.enable` is set to false
```

### CODE REVIEW

**Comment by upodroid** [comment]
:( 

Surprised there haven't been more reports of broken HTTPS ILBs

```
  # module.core_web_internal_lb.google_compute_region_health_check.default["default"] will be updated in-place
  ~ resource "google_compute_region_health_check" "default" {
        id                  = "projects/REDACTED/regions/europe-west1/healthChecks/core-web-internal-hc-default"
        name                = "core-web-internal-hc-default"
        # (9 unchanged attributes hidden)


      + log_config {
          + enable = false
        }
        # (1 unchanged block hidden)
    }

  # module.cseye_web_internal_lb.google_compute_region_backend_service.default["default"] will be updated in-place
  ~ resource "google_compute_region_backend_service" "default" {
        id                              = "projects/REDACTED/regions/europe-west1/backendServices/cseye-web-internal-backend-default"
        name                            = "cseye-web-internal-backend-default"
        # (14 unchanged attributes hidden)


      ~ log_config {
          - sample_rate = 1 -> null
            # (1 unchanged attribute hidden)
        }
        # (1 unchanged block hidden)
    }
```

File: mmv1/products/compute/terraform.yaml [hunk 1]
```diff
@@ -419,6 +419,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       timeoutSec: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+      logConfig: !ruby/object:Overrides::Terraform::PropertyOverride
```
[the remark this answers is not in the document]
Comment (upodroid) [reply mmv1/products/compute/terraform.yaml]: Fixed

**Comment by upodroid** [comment]
Bump, would be great to ship this ASAP. That diff is becoming is rather annoying

Will also fix TestAccComputeRegionBackendService_regionBackendService* failures

**Comment by upodroid** [comment]
@rileykarson Can I get this merged?

**Comment by upodroid** [comment]
Last commit fixes the following diff.

```
-----------------------------------------------------
2021/09/21 21:07:09 [DEBUG] Google API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 200 OK
Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000,h3-T051=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
Cache-Control: private
Content-Type: application/json; charset=UTF-8
Date: Tue, 21 Sep 2021 20:07:10 GMT
Etag: VLvcHogFWotRS8n_0kNMLs-bDdk=/PfRIcDZVB78QcDfTjJZeVeEP5Sc=
Server: ESF
Vary: Origin
Vary: X-Origin
Vary: Referer
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Xss-Protection: 0

{
  "id": "43336837416584631",
  "creationTimestamp": "2021-09-21T13:06:48.571-07:00",
  "name": "foo-i4zm6oenwa",
  "description": "",
  "selfLink": "https://www.googleapis.com/compute/v1/projects/REDACTED/regions/us-central1/backendServices/foo-i4zm6oenwa",
  "healthChecks": [
"https://www.googleapis.com/compute/v1/projects/REDACTED/global/healthChecks/bar-80cqhn6b69"
  ],
  "timeoutSec": 30,
  "port": 80,
  "protocol": "HTTP",
  "fingerprint": "laUj8vIMKmg=",
  "portName": "http",
  "sessionAffinity": "NONE",
  "affinityCookieTtlSec": 0,
  "region": "https://www.googleapis.com/compute/v1/projects/REDACTED/regions/us-central1",
  "loadBalancingScheme": "INTERNAL_MANAGED",
  "connectionDraining": {
    "drainingTimeoutSec": 0
  },
  "iap": {
    "enabled": true,
    "oauth2ClientId": "test",
    "oauth2ClientSecretSha256": "9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08"
  },
  "logConfig": {
    "enable": true,
    "sampleRate": 1
  },
  "localityLbPolicy": "RING_HASH",
  "consistentHash": {
    "httpCookie": {
      "name": "mycookie",
      "ttl": {
        "seconds": "11",
        "nanos": 1234
      }
    },
    "minimumRingSize": "1024"
  },
  "circuitBreakers": {
    "maxConnections": 10,
    "maxPendingRequests": 1024,
    "maxRequests": 1024,
    "maxRetries": 3
  },
  "outlierDetection": {
    "consecutiveErrors": 2,
    "maxEjectionPercent": 10,
    "enforcingConsecutiveErrors": 100,
    "enforcingSuccessRate": 100,
    "successRateMinimumHosts": 5,
    "successRateRequestVolume": 100,
    "successRateStdevFactor": 1900,
    "consecutiveGatewayFailure": 5
  },
  "kind": "compute#backendService"
}

-----------------------------------------------------
2021/09/21 21:07:09 [DEBUG] Retry Transport: Stopping retries, last request was successful
2021/09/21 21:07:09 [DEBUG] Retry Transport: Returning after 1 attempts
2021/09/21 21:07:13 [INFO] Authenticating using DefaultClient...
2021/09/21 21:07:13 [INFO]   -- Scopes: [https://www.googleapis.com/auth/compute https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/cloud-identity https://www.googleapis.com/auth/ndev.clouddns.readwrite https://www.googleapis.com/auth/devstorage.full_control https://www.googleapis.com/auth/userinfo.email]
2021/09/21 21:07:13 [INFO] Authenticating using DefaultClient...
2021/09/21 21:07:13 [INFO]   -- Scopes: [https://www.googleapis.com/auth/compute https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/cloud-identity https://www.googleapis.com/auth/ndev.clouddns.readwrite https://www.googleapis.com/auth/devstorage.full_control https://www.googleapis.com/auth/userinfo.email]
2021/09/21 21:07:13 [DEBUG] Waiting for state to become: [success]
2021/09/21 21:07:13 [INFO] Terraform is using this identity: mahamed.ali.ext@proximus.com
2021/09/21 21:07:13 [WARN] Truncating attribute path of 0 diagnostics for TypeSet
    provider_test.go:270: Step 1/4 error: After applying this test step, the plan was not empty.
        stdout:
        
        
        An execution plan has been generated and is shown below.
        Resource actions are indicated with the following symbols:
          ~ update in-place
        
        Terraform will perform the following actions:
        
          # google_compute_region_backend_service.foobar will be updated in-place
          ~ resource "google_compute_region_backend_service" "foobar" {
                id                              = "projects/REDACTED/regions/us-central1/backendServices/foo-i4zm6oenwa"
                name                            = "foo-i4zm6oenwa"
                # (15 unchanged attributes hidden)
        
        
        
        
              ~ log_config {
                  - sample_rate = 1 -> null
                    # (1 unchanged attribute hidden)
                }
        
                # (4 unchanged blocks hidden)
            }
        
        Plan: 0 to add, 1 to change, 0 to destroy.
2021/09/21 21:07:19 [INFO] Authenticating using DefaultClient...
```

**Comment by upodroid** [comment]
@rileykarson Now that v4 is out the door, can you take a look at this? This is a problem for me right now.

**Review by rileykarson — APPROVED** [review]

**Pull request merged**